README.md

```diff
@@ -96,22 +96,22 @@ explicitly support other platforms.
     - This project will automatically check if the compiler provides
     `std::filesystem`. If it is not found, then Boost Filesystem is required.
     This behavior can be controlled with the `VC_USE_BOOSTFS` CMake flag.
-* [Doxygen](http://www.doxygen.org/): Required to build
+* [Doxygen](https://github.com/doxygen/doxygen): Required to build
```

Comment:
```suggestion
* [Doxygen](https://www.doxygen.nl/): Required to build
```

Reply:
Using the Doxygen website since it's the official location for downloading binary releases.